integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeFunctionIT.java

```diff
@@ -636,7 +636,7 @@ public void testDateFormat() throws IOException {
     String timestamp = "1998-01-31 13:14:15.012345";
     String timestampFormat = "%a %b %c %D %d %e %f %H %h %I %i %j %k %l %M "
         + "%m %p %r %S %s %T %% %P";
-    String timestampFormatted = "Sat Jan 01 31st 31 31 12345 13 01 01 14 031 13 1 "
+    String timestampFormatted = "Sat Jan 01 31st 31 31 012345 13 01 01 14 031 13 1 "
```

Comment:
what is the change? 12345 -> 012345

Reply:
For `DATE_FORMAT` and `TIME_FORMAT` functions, microseconds should be padded with 0s on the left side according to both [MySQL](https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_date-format) and [our documentation](https://github.com/opensearch-project/sql/blob/main/docs/user/dql/functions.rst#date-format) (6 digits of precision should always be returned when using `%f`). This was fixed and the test was changed.